cves/CVE-2007-5000.yml

```diff
@@ -42,27 +45,17 @@ bounty:
   announced:
   url:
 reviews: []
-bugs: []
+bugs: [26838]
 repo:
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-   - commit: 39d3ff3144fae75422e0255df0f5225c95c1ceb8
-     note:
-   - commit: 8d8e3d02c9090768b7ca4d9bd552c28a16d48ac6
-     note:
-   - commit: 39d3ff3144fae75422e0255df0f5225c95c1ceb8
-     note:
-   - commit: b7881ae804aa98ca91cc47616b41fb7ece9f7c70
-     note:
-   - commit: 9de8a693d16e5ce76ff5b7f727311e783a51f34a
-     note: Part of a file rename
+   - commit: 5b442e5f1b9b81bfe8299259671ec43d3be129c3
+     note: This commit removed an XSS vulnerability by escaping the string.
```

Comment:
Was there a reason the existing fixes were removed? Were they wrong(just curious, I'm a researcher on this project.)

Reply:
I tried searching for these commit hashes and found nothing. I don't know where they came from.

Reply:
Turns out that when I search all of Github with these hashes, they do show up. However, they all seem to fix the same piece of code, just in different httpd repositories. I'll add them back in.